src/domain/harbors/harborDetails/HarborDetails.tsx

```diff
@@ -1,67 +1,87 @@
 import React from 'react';
-import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
 
 import Grid from '../../../common/grid/Grid';
 import ExternalLink from '../../../common/externalLink/ExternalLink';
-import LabelValuePair from '../../../common/labelValuePair/LabelValuePair';
 import { HarborData } from '../utils';
 import styles from './harborDetails.module.scss';
 import Section from '../../../common/section/Section';
+import Text from '../../../common/text/Text';
+import { formatDimension } from '../../../common/utils/format';
+import List from '../../../common/list/List';
+import ListItem from '../../../common/list/ListItem';
 
 type Props = {
+  maps: HarborData['maps'];
   imageFile: HarborData['imageFile'];
   streetAddress: HarborData['streetAddress'];
   zipCode: HarborData['zipCode'];
   municipality: HarborData['municipality'];
-  wwwUrl: HarborData['wwwUrl'];
   servicemapId: HarborData['servicemapId'];
   maxWidth: HarborData['maxWidth'];
 };
 
 const HarborDetails: React.SFC<Props> = ({
+  maps,
   imageFile,
   streetAddress,
   zipCode,
   municipality,
-  wwwUrl,
   servicemapId,
   maxWidth,
 }) => {
-  const { t } = useTranslation();
-  const address = `${streetAddress} ${zipCode} ${municipality}`;
+  const { t, i18n } = useTranslation();
+  const address = `${streetAddress}, ${zipCode} ${municipality}`;
   const imageSrc = imageFile ? imageFile : '';
-  const url = wwwUrl ? wwwUrl : '';
   const serviceMapUrl = `${process.env.REACT_APP_SERVICE_MAP_URI}${servicemapId}`;
 
   return (
-    <Grid colsCount={3}>
-      <div className={classNames(styles.section, styles.harborAddress)}>
+    <Grid colsCount={4}>
+      <div className={styles.column}>
         <img className={styles.image} src={imageSrc} alt="map" />
-        <div className={classNames(styles.address)}>
-          <Section title={t('harbors.details.address')}>
-            <ExternalLink href={url} variant="withArrow">
-              {address}
+      </div>
+      <div className={styles.column}>
+        <Section className={styles.address} title={t('common.terminology.address').toUpperCase()}>
+          {address}
+        </Section>
+        <Section>
+          <Text weight="bold">
+            <ExternalLink href={serviceMapUrl} variant="withArrow">
+              {t('common.terminology.serviceMap')}
             </ExternalLink>
-            <ExternalLink href="">{t('harbors.details.portMap')}</ExternalLink>
-            <ExternalLink href={serviceMapUrl}>{t('harbors.details.serviceMap')}</ExternalLink>
+          </Text>
+        </Section>
+        {maps.length > 0 && (
+          <Section>
+            <Text weight="bold">
```

Comment:
Could we simplify this by removing the `List` ?
```suggestion
            <Text weight="bold">
              {maps.length > 1 ? (
                maps.map((map: Map, index: number) => (
                  <ExternalLink href={map.url} variant="withArrow">
                    {`${t('common.terminology.harborMap')} ${index + 1} (PDF)`}
                  </ExternalLink>
                ))
              ) : (
                <ExternalLink href={maps[0].url} variant="withArrow">
                  {`${t('common.terminology.harborMap')} (PDF)`}
                </ExternalLink>
              )}
            </Text>
```

Reply:
ExternalLinks are inline elements, so this would break the appearance. It's possible if the external links are given a custom style to display them on new rows.

Reply:
Good point, you are correct. Then one possibility is also to add just one wrapper div like this:
```suggestion
                  <div>
                    <ExternalLink href={map.url} variant="withArrow">
                      {`${t("common.terminology.harborMap")} ${index + 1} (PDF)`}
                    </ExternalLink>
                  </div>
```
this is not a biggie, you can decide :) I'm fine with the current approach also

Reply:
Suggestion taken 👍